scarlett/basics/talk.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: UTF-8 -*-
+
+# NOTE: Borrowed a great deal of knowledge from LISA project.
+
+import gobject
+gobject.threads_init()
+
+# Imports
+import scarlett
+import threading
+import os
+import gettext
+from Queue import Queue
+from time import sleep
+from subprocess import call
+# import urllib
+# from urllib import urlencode, urlopen
+from random import randint
+# from twisted.python import log
+from scarlett.basics import *
+from scarlett.constants import *
+import scarlett.basics.voice
+
+# System utterance definition : key : {(weight1, message1), (weight2, message2)}
+# Talk will randomly choose a message in list, balanced by weights
+_utterances = {
```

Comment:
One thing I realized i'll need to do to make my dream come true with this `ScarlettTalk` class ... going to need to upgrade to use Gstreamer 1.0 instead of 0.10. So I think i'm going to keep this PR small ... merge, and start working on installing / porting Gstreamer 0.10 -> 1.0. Then ill be able to use a nifty espeak plugin that has been added in the 1.0 series to control all TTS!